manuscript/060_Statement_First_Programming.md

```diff
@@ -110,13 +110,13 @@ public void ShouldRecognizeTimeSlotAboveMaximumAllowedAsInvalid()
 }
 ```
 
-Note how the method `PerformForTimeSlotIn()`, which triggers the specified behavior is accidentally called BEFORE the mock is actually set up and the set up return value is never taken into account. By some strange coincidence, this error did not alter the expected end result so we did not even notice. It sometimes turns out like this, most often in case of various boundary values (nulls etc.).
+Note how the method `PerformForTimeSlotIn()`, which triggers the specified behavior, is accidentally called *before* the mock is actually set up and the value of `frame.GetTimeSlot_Returns` is not taken into account. Thus this erroneous value does not alter the expected end result, but we'll not notice it. It sometimes turns out like this, most often in case of various boundary values (nulls etc.).
```

Comment:
I slightly preferer "go unnoticed", although both seem correct. 

The only *reason* I can come up with is that the word 'pass' is already (strongly) connected to tests passing and failing...